### PR TITLE
CocoaPods integration

### PR DESCRIPTION
**Pull request by Frugghi**
Implementation of all the required methods for CocoaPods integration.

### CODE REVIEW

File: routes/originPod.js [hunk 1]
```diff
@@ -6,25 +6,33 @@ const router = require('express').Router()
...
 router.get(
   '/:name',
   asyncMiddleware(async (request, response) => {
     const { name } = request.params
-    const url = `https://cocoapods.org/api/v1/pods?per_page=100&q=${name}`
-    const answer = await requestPromise({ url, method: 'GET', json: true })
-    const result = answer.crates.map(x => {
+    const algolia = {
+        appID: 'WBHHAMHYNM',
+        apiKey: '4f7544ca8701f9bf2a4e55daff1b09e9'
```
Comment (craigez) [inline routes/originPod.js]: Should this key be fetched from the configuration?

Comment (Frugghi) [reply routes/originPod.js]: Good question, this API key is managed by the CocoaPods admins and it's not something we can generate/replace. I took it from [here](https://github.com/CocoaPods/cocoapods.org/blob/bcd28fe68cd82390e0f66c9988b9e109eed58695/assets/javascripts/search.config.js#L689).

File: providers/summary/clearlydefined.js [hunk 2]
```diff
@@ -138,9 +138,12 @@ class ClearlyDescribedSummarizer {
   }
 
   addPodData(result, data) {
-    // setIfValue(result, 'described.releaseDate', extractDate(data.releaseDate))
-    // const license = SPDX.normalize(get(data, 'registryData.license'))
-    // setIfValue(result, 'licensed.declared', license)
+    setIfValue(result, 'described.releaseDate', extractDate(data.releaseDate))
+    setIfValue(result, 'described.projectWebsite', get(data, 'registryData.homepage'))
```
Comment (dabutvin) [inline providers/summary/clearlydefined.js]: 👍

**Comment by dabutvin** [comment]
great 👍 👍 👍

**Pull request merged**